Sources/Orbit/Support/BindingWrapper.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+/// A wrapper that provides a binding to its content.
+///
+/// This binding can either be supplied, in which case it is used directly,
+/// or one is derived from internal state (starting with `defaultValue`.
+///
+/// This is is useful for components that can manage their own state,
+/// but we also want to make it possible for that state to be driven
+/// from the outside if a binding is passed.
+struct BindingWrapper<Value, Content: View>: View {
```

Comment:
Our other wrappers are public - should we expose this one too? If yes, it would need to be added to docs ignore section

Reply:
Now I see this one is meant for something else than previews. 

Not sure if it should have a more distinct name, as now it looks almost like a duplicate of `StateWrapper`.